src/main/java/org/apache/commons/configuration2/ImmutableConfiguration.java

```diff
@@ -62,6 +62,20 @@ public interface ImmutableConfiguration {
      */
     boolean containsKey(String key);
 
+    /**
+     * Returns true if this configuration contains one or more matches to this value. This operation stops at first
+     * match but may be more expensive than the {@link #containsKey containsKey} method.
+     *
+     * @param value value whose presence in this configuration is to be tested
+     * @return {@code true} if this configuration maps one or more keys to the specified value
+     * @throws NullPointerException          if the value is {@code null}
+     * @throws UnsupportedOperationException if this method is not supported by the implementing class.
+     */
+    default boolean containsValue(String value) {
```

Comment:
Can this method be implemented in terms of other interface methods instead of throwing an exception?

Reply:
Ok, so I gave an actual default implementation to `containsValue`  in `ImmutableConfiguration` and tested it with that DummyClass that I didn't want to create to test it and see if it worked.

And it worked.

![image](https://github.com/apache/commons-configuration/assets/76500344/4604c968-fac6-4e6e-9779-a8e3524b54db)

![image](https://github.com/apache/commons-configuration/assets/76500344/869a88d0-ce77-4e7f-96c4-561cb50bd681)

![image](https://github.com/apache/commons-configuration/assets/76500344/206f1ca3-9a6b-403f-941f-8b357eb8c8bd)